packages/styletron-engine-atomic/src/css.js

```diff
@@ -17,11 +17,22 @@ export function keyframesToBlock(keyframes: {[string]: Object}): string {
   if (__DEV__) {
     validateKeyframesObject(keyframes);
   }
+  if (__DEV__ && typeof Object.getPrototypeOf(keyframes) !== "undefined") {
+    if (Object.getPrototypeOf(keyframes) !== Object.getPrototypeOf({})) {
+      // eslint-disable-next-line no-console
+      console.warn(
+        "Only plain objects should be used as animation values. Unexpectedly recieved:",
+        keyframes,
+      );
+    }
+  }
   let result = "";
   for (const animationState in keyframes) {
-    result += `${animationState}{${declarationsToBlock(
-      keyframes[animationState],
-    )}}`;
+    if (Object.prototype.hasOwnProperty.call(keyframes, animationState)) {
```

Comment:
I'm wondering whether this runtime check is required if we have development warnings. Maybe just the development warnings will suffice?

Everywhere else in styletron it is assumed everything is always a POJO.

Reply:
that sounds good to me